.changeset/few-rules-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/sdk": minor
```

Comment:
Did you mean
```suggestion
"@directus/sdk": major
```
?

Reply:
I did not. While it definitely *can* be considered a breaking change i don't personally think it is. Feel free to update to major tho if you disagree. `patch` was definitely not correct.

If we're going with a major update here tho we may want to bundle it with https://github.com/directus/directus/pull/19354 which can also potentially be seen as breaking change (since function arguments changed)

Reply:
Lets bundle those changes and release it as a single breaking change 👍🏻 

I prefer having strict semver wherever we can 🙂

Reply:
Boundaries are thin here 😄 But ultimately I also think a breaking change is appropriate because of the structure change 👍

Reply:
We change the IO of a function without being backwards compatible, it's a breaking change! If it was backwards compatible, it would've been a minor 👍🏻

Reply:
Haha yeah, you're absolutely right! 😇 Just need to put the brain in `SemVer` mode... 🙈